ark/segmentation/marker_quantification.py

```diff
@@ -21,21 +22,36 @@ def compute_marker_counts(input_images, segmentation_masks, nuclear_counts=False
             rows x columns x compartment matrix of masks
         nuclear_counts (bool):
             boolean flag to determine whether nuclear counts are returned
+        regionprops_features (list):
+            morphology features for regionprops to extract for each cell
 
     Returns:
         xarray.DataArray:
             xarray containing segmented data of cells x markers
     """
 
-    unique_cell_ids = np.unique(segmentation_masks[..., 0].values)
+    if regionprops_features is None:
+        regionprops_features = ['label', 'area', 'eccentricity', 'major_axis_length',
+                                'minor_axis_length', 'perimeter', 'centroid']
+
+    if 'coords' not in regionprops_features:
+        regionprops_features.append('coords')
+
+    # create variable to hold names of returned columns only
+    regionprops_names = copy.copy(regionprops_features)
+    regionprops_names.remove('coords')
 
-    # define morphology properties to be extracted from regionprops
-    object_properties = ["label", "area", "eccentricity", "major_axis_length",
-                         "minor_axis_length", "perimeter", 'coords']
+    # centroid returns two columns, need to modify names
+    if np.isin('centroid', regionprops_names):
+        regionprops_names.remove('centroid')
+        regionprops_names += ['centroid-0', 'centroid-1']
```

Comment:
I don't see where this split is being used,  Is it being used implicitly in the background?

Reply:
If you supply 'centroid' to reginonprops_table, it returns columns with those names for each part of the centroid.